### PR TITLE
clean docs of luis:build and qna build and cross-train

### LINKED ISSUE #981: Docs for luis/qnamaker:build is not align with the latest changes
## Versions
4.10.1

## Describe the bug
Docs for luis/qnamaker:build is not updated to latest that is confusing users.

## Expected behavior
Update the doc and give users the latest usage information.

## Screenshots
image

## Additional context

[bug]

### PR DESCRIPTION
**Pull request by feich-ms**
fix #981 to clean docs of luis:build, qnamaker:build, luis:cross-train, qnamaker:cross-train

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -736,28 +736,27 @@ USAGE
...
-  -f, --force                      If --dialog flag is provided, overwrites relevant dialog file
-  -h, --help                       show CLI help
+  -f, --force                      If --out flag is provided with the path to an existing file, overwrites that file
+  -h, --help                       luis:build command help
   -i, --in=in                      Lu file or folder
 
-  -o, --out=out                    Output file or folder name. If not specified, current directory will be used as
-                                   output
-
+  -o, --out=out                    Output folder name to write out .dialog files. If not specified, application
```
Comment (boydc2014) [inline packages/cli/README.md:743]: .dialog and .settings.json files?

File: packages/cli/README.md [hunk 2]
```diff
@@ -766,9 +765,15 @@ OPTIONS
...
+  --endpoint                       Luis authoring endpoint for publishing
+
+  --schema=schema                  Defines $schema for generated .dialog files
+
+  --isStaging                      Publishes luis application to staging slot if set. Default to production slot
+
 EXAMPLE
 
-       $ bf luis:build --in {INPUT_FILE_OR_FOLDER} --authoringKey {AUTHORING_KEY} --botName {BOT_NAME} --dialog 
+       $ bf luis:build --in {INPUT_FILE_OR_FOLDER} --authoringKey {AUTHORING_KEY} --botName {BOT_NAME}
```
Comment (boydc2014) [inline packages/cli/README.md:776]: -- dialog is missing?

File: packages/cli/README.md (hunk 1 above)
Comment (feich-ms) [reply packages/cli/README.md]: fixed

File: packages/cli/README.md (hunk 2 above)
Comment (feich-ms) [reply packages/cli/README.md:776]: fixed

**Comment by feich-ms** [comment]
@boydc2014 comments resolved.

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**